tools/ci_build/github/apple/package_assembly_utils.py

```diff
@@ -12,11 +13,27 @@
 _script_dir = pathlib.Path(__file__).parent.resolve(strict=True)
 repo_root = _script_dir.parents[3]
 
+
+class PackageVariant(enum.Enum):
+    Full = 0  # full ORT build
+    Mobile = 1  # minimal ORT build with reduced ops
```

Comment:
perhaps there's a better name than "Mobile", this is just based on the name of the existing packages (e.g., onnxruntime-mobile-c)
"Minimal"? suggestions are welcome

Reply:
I think Mobile is consistent with all the other messaging/doco we have about the minimal build with ops/types based on common mobile models.